compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/OpOracle.cpp

```diff
@@ -165,15 +165,6 @@ bool isHoistableConstExprLeaf(const ConstExprAnalysis::ConstValueInfo *info) {
     }
   }
 
-  // Never hoist sub-byte aligned values: in legal programs, these will be
-  // cast or packed in some successor.
```

Comment:
This still needs a better way to convey some sort of contract w.r.t. what is valid to hoist, given that this PR is relying on bitcasting defined with a user hook rather than on the analysis itself.